assets/dev/js/frontend/handlers/video.js

```diff
@@ -66,10 +71,79 @@ export default class Video extends elementorModules.frontend.handlers.Base {
 		this.getLightBox().setVideoAspectRatio( this.getElementSettings( 'aspect_ratio' ) );
 	}
 
+	prepareYTVideo( YT, onOverlayClick ) {
+		const elementSettings = this.getElementSettings(),
+			playerOptions = {
+				videoId: this.videoID,
+				events: {
+					onReady: () => {
+						if ( elementSettings.mute ) {
+							this.player.mute();
+						}
+
+						if ( elementSettings.autoplay || onOverlayClick ) {
+							this.player.playVideo();
+						}
+					},
+					onStateChange: ( event ) => {
+						if ( event.data === YT.PlayerState.ENDED && elementSettings.loop ) {
+							this.player.seekTo( elementSettings.start || 0 );
+						}
+					},
+				},
+				playerVars: {
+					controls: elementSettings.controls ? 1 : 0,
+					rel: elementSettings.rel ? 1 : 0,
+					playsinline: elementSettings.play_on_mobile ? 1 : 0,
+					modestbranding: elementSettings.modestbranding ? 1 : 0,
+					autoplay: elementSettings.autoplay ? 1 : 0,
+					start: elementSettings.start,
+					end: elementSettings.end,
```

Comment:
It's needed? it looks like the start/end are set in the next lines.

Reply:
Yes, this is actually needed and the next lines are not, will remove them.

Reply:
Done.